xml/docker_building_images.xml

```diff
@@ -74,7 +74,7 @@
   </para>
 
   <sect2>
-   <title>Optaining Base images of &slea;&nbsp;12 SP3 and later Service Packs</title>
```

Comment:
Thanks, Fabian! Could you also change 'later' to 'Later' and 'image' to 'Image' in both titles, please? See https://doc.opensuse.org/documentation/styleguide/index.html#sec.capitalization